### PR TITLE
Update: fix typos in 03-openai-assistants.mdx

### PR DESCRIPTION
**Pull request by draganaleksich**

### CODE REVIEW

**Comment by DraganAleksic99** [comment]
I think that first part of the Configure Request Options section is not correct. By default, useAssistant hook does not send a request to /api/completion endpoint, the api property of useAssistantOptions is a required property :)

**Review by MISTYKITTY — COMMENTED** [review]
****

File: content/docs/05-ai-sdk-ui/03-openai-assistants.mdx [hunk 1]
```diff
@@ -202,4 +202,4 @@ const { messages, input, handleInputChange, handleSubmit } = useAssistant({
 });
 ```
 
-In this example, the `useAssistant` hook sends a POST request to the `/api/completion` endpoint with the specified headers, additional body fields, and credentials for that fetch request. On your server side, you can handle the request with these additional information.
+In this example, the `useAssistant` hook sends a POST request to the `/api/custom-completion` endpoint with the specified headers, additional body fields, and credentials for that fetch request. On your server side, you can handle the request with these additional information.
```
Comment (lgrammel) [inline content/docs/05-ai-sdk-ui/03-openai-assistants.mdx:205]: It's `/api/assistant` (line 19)

**Review by lgrammel — CHANGES_REQUESTED** [review]
Thanks! found one small mistake

File: content/docs/05-ai-sdk-ui/03-openai-assistants.mdx (hunk 1 above)
Comment (DraganAleksic99) [reply content/docs/05-ai-sdk-ui/03-openai-assistants.mdx:205]: Done!

**Review by lgrammel — APPROVED** [review]

**Comment by lgrammel** [comment]
Great thank you!

**Comment by DraganAleksic99** [comment]
You're welcome!

**Pull request merged**